api/api/utils/image_proxy/extension.py

```diff
@@ -47,7 +46,7 @@ async def get_image_extension(image_url: str, media_identifier) -> str | None:
 
             await _cache_extension(cache, key, ext)
         except Exception as exc:
-            sentry_sdk.capture_exception(exc)
+            logger.error("upstream_thumbnail_exception", e=exc)
```

Comment:
I don't think this works as expected, or at least, it doesn't work the way `structlog-sentry` wants you to pass exc info:

> This won't automatically collect sys.exc_info() along with the message, if you want to enable this behavior, just pass exc_info=True.

I think it might be this that we need instead?

```suggestion
            logger.error("upstream_thumbnail_exception", exc_info=True)
```